frontend/src/metabase/admin/settings/components/CloudPanel/CloudPanel.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import { t } from "ttag";
+
+import {
+  useGetCloudMigrationQuery,
+  useCreateCloudMigrationMutation,
+} from "metabase/api";
+import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
+import { useDispatch } from "metabase/lib/redux";
+import { refreshSiteSettings } from "metabase/redux/settings";
+import { Box, Text } from "metabase/ui";
+
+import { MigrationError } from "./MigrationError";
+import { MigrationInProgress } from "./MigrationInProgress";
+import { MigrationStart } from "./MigrationStart";
+import { MigrationSuccess } from "./MigrationSuccess";
+import {
+  type InternalCloudMigrationState,
+  getCheckoutUrl,
+  isInProgressMigration,
+} from "./utils";
+import { getStartedVisibleStates, pollingIntervalsByState } from "./utils";
+
+export const CloudPanel = () => {
+  const dispatch = useDispatch();
+  const [pollingInterval, setPollingInterval] = useState<number | undefined>(
+    undefined,
+  );
+
+  const {
+    data: migration,
+    isLoading,
+    error,
+  } = useGetCloudMigrationQuery(undefined, {
+    refetchOnMountOrArgChange: true,
```

Comment:
RTK Query has a `skipPollingIfUnfocused` flag we could use to avoid making network requests while the user is on other windows. Might be handy, but not something that's a show stopper either

Reply:
This requires that we have RTKQuery configured globally to do this (see [setupListeners](https://redux-toolkit.js.org/rtk-query/api/setupListeners)). I'll do this as a follow up, I've been wanting to add this to the admin logs page as well.